src/pkg/cli/client/byoc/baseclient.go

```diff
@@ -92,7 +92,7 @@ func DebugPulumi(ctx context.Context, env []string, cmd ...string) error {
 	// Run the Pulumi command locally
 	env = append([]string{
 		"PATH=" + os.Getenv("PATH"),
-		"USER=" + os.Getenv("USER"), // needed for Pulumi
+		"USER=" + pkg.GetCurrentUser(), // needed for Pulumi
```

Comment:
Would this results in Unicode usernames? And would it be of any issues?

Reply:
UTF-8,should be fine